.pipelines/pipeline.yaml

```diff
@@ -377,94 +331,45 @@ stages:
     parameters:
       name: "cilium_e2e"
       displayName: Cilium
-      testDropgz: ""
       clusterType: swift-byocni-nokubeproxy-up
       clusterName: "ciliume2e"
       vmSize: Standard_B2ms
       k8sVersion: ""
-      dependsOn: "containerize"
-
-  - template: singletenancy/cilium/cilium-e2e-job-template.yaml
-    parameters:
-      name: "cilium_podsubnet_cni_e2e"
-      displayName: CNI - Cilium Podsubnet
-      testDropgz: true
-      clusterType: swift-byocni-nokubeproxy-up
-      clusterName: "cilpodcnie2e"
-      vmSize: Standard_B2ms
-      k8sVersion: ""
-      dependsOn: "containerize"
+      dependsOn: "test"
```

Comment:
nice! this makes the pipeline much more efficient. what happens if the create cluster step completes before the publish stage? does the e2e test step wait?

Reply:
yes, the e2e stages depend on the publish step, so it would wait there